Docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cat > .env  <<-EOF
+APP_KEY=${APP_KEY:-SomeRandomKey!}
+APP_ENV=${APP_ENV:-local}
+APP_DEBUG=${APP_ENV:-true}
+APP_LOCALE=${APP_LOCALE:-en}
+APP_FALLBACK_LOCALE=${APP_FALLBACK_LOCALE:-en}
+
+DB_CONNECTION=${DB_CONNECTION:-mysql}
+DB_HOST=${DB_PORT_3306_TCP_ADDR:-localhost}
```

Comment:
@SolidNerd could you use my way of discovering environment variables (see https://github.com/outcoldman/docker-dash-annotations/blob/master/entrypoint.sh) so basically if something is set with `-e` use it, if not - try to discover default images like `mariadb` or `mysql`